docs/plugins/composable_resources.md

```diff
@@ -197,6 +197,13 @@ To modify an existing passthru resource to be written to only after other eligib
 irods@hostname:~/ $ iadmin modresc newResc context 'write=0.4;read=1.0'
 ```
 
+Setting a `read` or `write` weight to `0` will prevent that action from occurring on the branch of the resource tree under the passthru (because a `0` vote signals that it cannot service the incoming request).  The following will 'turn off' the resource `newResc` (a policy-based way of marking a resource "down"):
```

Comment:
Could replace the single quotes with `**` so that the text is bold?
Or replace the single quotes with `_` so that the text is italic?

Reply:
could be - it was all consistent at the 4.2.0 release... and then, things got 'more'.

i'll stare it again.   i kind of like the 'code' treatment of the backticks...

Reply:
No no, I'm only referring to the `'turn of'` phrase.

Everything else is using backticks.

Reply:
ohhhh, yes - I'll try double quotes instead of single, first.

Reply:
doubles seems good